viewer/lib/components/spec_file.dart

```diff
@@ -187,41 +185,40 @@ class _SpecFileCardState extends State<SpecFileCard> {
             child: Column(
               crossAxisAlignment: CrossAxisAlignment.start,
               children: [
-                Row(
-                    mainAxisSize: MainAxisSize.max,
-                    children: [
-                      PanelNameRow(name: specManager.value.filename),
-                      IconButton(
-                        color: Colors.black,
-                        icon: Icon(Icons.open_in_new),
-                        tooltip: "Viewer",
-                        onPressed: () {
-                          if(rendererServiceAddress != "SPEC_RENDERER_SERVICE") {
-                            launch(rendererServiceAddress +"/" + specManager.value.name );
-                          } else {
-                            AlertDialog alert = AlertDialog(
-                              content: Text("Spec renderer service not configured"),
-                              actions: [
-                                TextButton(
-                                  child: Text("OK"),
-                                  onPressed: () {
-                                    Navigator.of(context).pop(); // dismiss dialog
-                                  },
-                                ),
-                              ],
-                            );
-                            // show the dialog
-                            showDialog(
-                              context: context,
-                              builder: (BuildContext context) {
-                                return alert;
+                Row(mainAxisSize: MainAxisSize.max, children: [
+                  PanelNameRow(name: specManager.value.filename),
+                  IconButton(
+                    color: Colors.black,
+                    icon: Icon(Icons.open_in_new),
+                    tooltip: "Viewer",
+                    onPressed: () {
+                      var address = rendererServiceAddress();
```

Comment:
Apart from the formatting change, the only difference here is that renderServiceAddress() is a function call and not a variable